test/pretest.c

```diff
@@ -25,7 +25,12 @@ int main(int argc, char *argv[])
     Uint64 prequit;
     (void)argc;
     (void)argv;
-    SDL_Init(0);
+    if (!SDL_Init(0))
```

Comment:
Braces of an if case don't need an extra line.

Reply:
Will fix.